src/RequestAPI.ts

```diff
@@ -10,6 +11,10 @@ export enum Rtype {
     music = "music",
     character = "character",
 }
+export enum Frametype {
```

Comment:
I don't think an extra Frametype is neccessary. If an 'IFrame` has non-null `branch`, we know it is a `divergence` frame